### PR TITLE
Add error handling for missing `js/_website/version.json`

### PR DESCRIPTION
**Pull request by hannahblair**
## Description

When running `npm i` from root, you'll see an error related to a missing module `version.json` in `js/_website/src/lib/json/`. This is because we need to run `generate_jsons/generate.py` to get that file, but if devs aren't touching any files related website, they shouldn't have to worry about this. This fix dynamically imports `version.json`, catches the error if it doesn't exist, and uses a fallback version (set to 4.0.0 - should this change?).

### CODE REVIEW

**Review by pngwn — APPROVED** [review]
Looks good. I think 4.0 is fine as a fallback. Thanks @hannahblair!

**Comment by aliabd** [comment]
the changes look fine to me but i can't figure out why website build is failing

**Comment by aliabd** [comment]
@hannahblair it seems to be related to the dynamic import. you need `.then((module) => module.default)` because it returns `_version`  as `{ default: { version: '4.19.0' } }` 

but that actually didn't solve the problem. version is still undefined when there is a version.json, and i'm not sure why. Try debugging with pnpm dev or pnpm build

**Review by pngwn — APPROVED** [review]
Looks good! Thanks @hannahblair!

**Comment by aliabd** [comment]
Nice @hannahblair the build errors are gone! but why did you remove `/3.50.2/docs` and `/3.50.2/guides` from the prerender routes? They cause the 404 error when you toggle to that version in docs and guides.

Bringing those back fixes everything though! nice work, this was an annoying bug.

File: js/_website/svelte.config.js [hunk 1]
```diff
@@ -23,8 +52,6 @@ const config = {
 				`/main/docs/js`,
 				`/main/docs/js/storybook`,
 				`/main/docs/js/`,
```
Comment (aliabd) [inline js/_website/svelte.config.js:54]:
```suggestion
				`/main/docs/js/`,
				`/3.50.2/docs`,
				`/3.50.2/guides`,
```

**Review by aliabd — APPROVED** [review]
lgtm aside from the 3.50.2 stuff!

**Pull request merged**